src/models/user.js

```diff
@@ -0,0 +1,17 @@
+import PomoTimer from './pomo-timer';
+
+class Model {
+  constructor(snapshot) {
+    this.snapshot = snapshot;
+    this.attrs = snapshot.val();
+  }
+  get ref() {
+    return this.snapshot.ref();
+  }
+}
+
+export default class User extends Model {
+  get pomo() {
+    return new PomoTimer(this.attrs.pomo);
+  }
+}
```

Comment:
Interestingly, these classes aren't used. Keeping them for reference if we do want a model class.